test/integration/belongsToMany.test.js

```diff
@@ -327,7 +455,7 @@ describe('belongsToMany', function () {
           as: 'awesomeMembers',
           through: 'project_members',
           foreignKey: 'projectId',
-          targetKey: 'userId',
+          targetKey: 'id',
```

Comment:
This does not look right to me, don't foreignKey and targetKey define the names on the join table?

Reply:
According to [Sequelize docs](https://sequelize.org/master/manual/assocs.html#for--code-belongstomany--code--relationships), this creates a junction table `project_members` with fields `projectId` (because there is no `sourceKey` here) and `userId` (because we are using `id` as the `targetKey`).

Reply:
Ah, right you are. Think i even wrote some of the code for targetKey haha.